assets/js/feedback.js

```diff
@@ -0,0 +1,48 @@
+  $(function($) {
+    $('#feedback').on('click',function(e){
+    e.preventDefault();
+    e.stopPropagation();
+    $('#feedbackForm textarea, #feedbackForm input:not([type="submit"])').val('');
+    $('#feedbackForm').modal({show:true});
+  });
+  $('#feedbackForm [id]').on('focus', function(e) {
+    if ($(e.currentTarget).parent().hasClass('has-error'))
+      $(e.currentTarget).parent().removeClass('has-error').find('span').css('visibility', 'hidden');
+  });
+  $('#feedbackForm').on('shown.bs.modal', function() {
+    $('#feedbackForm textarea').focus();
+  });
+
+  $('#feedbackForm input[type="submit"]').on('click', function (e) {
+    var valid = true;
+    $('#feedbackForm [id]').each(function (i, v) {
+      if ($(v).val() === '') {
+        $(v).parent().addClass('has-error').find('span').css('visibility', 'visible');
+        valid = false;
+      }
+      else
+        if ($(v).attr('id') === 'feedback-email' && !validateEmail($(v).val())) {
+          $(v).parent().addClass('has-error').find('span').css('visibility', 'visible');
+          valid = false;
+        }
+    });
+    if (valid)
+      $.ajax({
+        url   : 'https://oceansMap2.asascience.com/?SiteID=1&Name=' + $('#feedback-name').val() + '&Email=' + $('#feedback-email').val() + '&Comment=' + $('#feedback-comment').val(),
```

Comment:
@gitchrisadams I don't know the first thing about JS but my guess is that the message posted on this domain, right?

For this site @jbosch-noaa originally decided to have all the feedback as GitHub issues (top left banner). I don't really have a preference so I'll leave this one for @jbosch-noaa to decided and merge.

Reply:
Yes, message posted using that url. It matches how the other feedback buttons work for the ioos.us sites.